Automattic/wp-desktop
# Documentation: Correct a secrets link.

This PR corrects the link to detailed instructions for generating secrets required for development.

**Testing Instructions**
Verify the updated link in `docs/install.md` takes you to `docs/secrets.md`, instead of a GitHub 404.
